Dockerfile.rhel7

```diff
@@ -0,0 +1,34 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+WORKDIR /go/src/github.com/openshift/api
+COPY . .
+ENV GO_PACKAGE github.com/openshift/api
+RUN make build --warn-undefined-variables
```

Comment:
Is this the usage of build machinery go? If this is all we are using it for, I'm sure we can come up with our own build target to avoid importing it as a dependency